labs/CompositionCollectionView/src/CompositionCollectionView.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#nullable enable
+
+#if !WINAPPSDK
```

Comment:
You shouldn't need to special case the whole component like this or include any of the namespaces, that's what the global files are for.

Reply:
Yeah, I was delaying that change to try to minimize changes to the expressionnodes fork. But I gave it a try and it wasn't that bad. The library should be good now and I'll clean up the samples in my next pass.